priceoyeclone/urls.py

```diff
@@ -1,7 +1,12 @@
 from django.contrib import admin
+from django.contrib.auth import views as auth_views
 from django.urls import include, path
 
 urlpatterns = [
     path("admin/", admin.site.urls),
-    path("Users/", include("Users.urls")),
+    path("users/", include("users.urls")),
+    path('password_reset/', auth_views.PasswordResetView.as_view(), name='password_reset'),
+    path('password_reset/done/', auth_views.PasswordResetDoneView.as_view(), name='password_reset_done'),
```

Comment:
I noticed that you have implemented the password reset flow and these URLs can be moved in users app.